### ISSUE TITLE
Wrong path for Porcupine libraries in 2.5.8 docker image

### ISSUE THREAD

**Issue by heyryanw** [rhasspy/rhasspy#155]
Hardware: Raspberry Pi 4b
OS: Raspios arm64

While trying to use Porcupine in the 2.5.8 Docker image, I got this error:
```
OSError: /usr/lib/rhasspy/rhasspy-wake-porcupine-hermes/rhasspywake_porcupine_hermes/porcupine/lib/linux/x86_64/libpv_porcupine.so: cannot open shared object file: No such file or directory
2020-12-26 09:25:09,531 INFO exited: wake_word (exit status 1; not expected)
2020-12-26 09:25:10,533 INFO gave up: wake_word entered FATAL state, too many start retries too quickly
[ERROR:2020-12-26 09:25:32,557] rhasspyserver_hermes:
Traceback (most recent call last):
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1821, in full_dispatch_request
    result = await self.dispatch_request(request_context)
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1869, in dispatch_request
    return await handler(**request_.view_args)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 792, in api_wake_words
    hotwords = await core.get_hotwords()
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__init__.py", line 887, in get_hotwords
    handle_finished(), messages, message_types
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__init__.py", line 971, in publish_wait
    result_awaitable, timeout=timeout_seconds
  File "/usr/lib/python3.7/asyncio/tasks.py", line 449, in wait_for
    raise futures.TimeoutError()
concurrent.futures._base.TimeoutError
```

Looking into the container a bit, I see the shared lib where it is expected to be, and the file is accesible. I also attempted to edit the profile.json file and use the 'library_path' directive as[ referenced here](https://rhasspy.readthedocs.io/en/latest/reference/#profile-settings) to instead point to the cortex-a72 version (this is running on a Rpi 4b), but the change isn't reflected when Rhasspy is restarted. Any ideas?

**Comment by codewing** [comment]
Related: I think I'm facing the same issue https://community.rhasspy.org/t/wake-word-porcupine-help-is-needed/2104/3

**Comment by synesthesiam** [comment]
Should be fixed in 2.5.9. Porcupine has been updated to v1.9, and the `pvporcupine` package is now used (which should do platform detection internally).

Please re-open the issue if the problem persists. Thanks!
